src/ecmascript_simd.js

```diff
@@ -120,6 +120,12 @@ function simdCreate(type) {
   return type.fn.apply(type.fn, lanes);
 }
 
+function simdCreateFromBuffer(type, buffer) {
+  var newValue = type.fn.apply(type.fn, lanes);
+  newValue.s_ = new type.view(buffer);
```

Comment:
Yes, it would, but that OK, I think.  SIMD values are immutable soit's impossible to create new values without creating new objects, in other words you cannot change the value of an existing SIMD value object.  For example, if you try to change a lane value (with replaceLane) you'll get a new object.